src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -83,12 +81,6 @@ func (r *Cluster) ValidateUpdate(old runtime.Object) error {
 				r.Spec.Replicas,
 				"scaling down is not supported"))
 	}
-	if !reflect.DeepEqual(r.Spec.Configuration, oldCluster.Spec.Configuration) {
```

Comment:
With this change the test description need to be updated

Remove: 

> (unable to update configuration)

- https://github.com/vectorizedio/redpanda/pull/822/commits/4de256127edbd25ff462ac4caf19f03a07f76352#diff-46bd3dd552497ca6d98b48ef0c41da598b2090731a86a4880c46d94d76c5bb7dR121
- https://github.com/vectorizedio/redpanda/pull/822/commits/4de256127edbd25ff462ac4caf19f03a07f76352#diff-46bd3dd552497ca6d98b48ef0c41da598b2090731a86a4880c46d94d76c5bb7dR131
- https://github.com/vectorizedio/redpanda/pull/822/commits/4de256127edbd25ff462ac4caf19f03a07f76352#diff-46bd3dd552497ca6d98b48ef0c41da598b2090731a86a4880c46d94d76c5bb7dL142

Reply:
oh okay, I missed that, thanks

Reply:
The tests should be ok, but the description is off.